app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new; end
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    redirect_to posts_path if @comment.save
```

Comment:
What happens if a comment is not saved successfully? Maybe you can add code to address that scenario, and test for it first in `comments_controller_spec` [OPTIONAL]

Same applies for like

Reply:
@Torres-ssf, see this :point_up:  :thinking: